docs/docs/index.md

```diff
@@ -16,9 +16,14 @@ Poetry provides a custom installer that will install `poetry` isolated
 from the rest of your system by vendorizing its dependencies. This is the
 recommended way of installing `poetry`.
 
+### osx / linux / bashonwindows install instructions
 ```bash
 curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
 ```
+### windows powershell install instructions
+```powershell
+$(Invoke-WebRequest -Uri https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py).Content | python
```

Comment:
```suggestion
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py -UseBasicParsing).Content | python
```

Reply:
From the Microsoft docs:

`-UseBasicParsing`

> This parameter has been deprecated. Beginning with PowerShell 6.0.0, all Web requests use basic parsing only. 
> This parameter is included for backwards compatibility only and any use of it has no effect on the operation of the cmdlet.

Reply:
Also the dollar sign is required so powershell knows to actually invoke the cmdlet to pass the result to the pipeline

Reply:
The problem is that Windows 10 and Windows Server 2019 ship with PowerShell 5.1 as the default version, not PowerShell 6, so the original command does not work for most users.

About the dollar sign, it is _not_ required, I have just retested and successfully installed Poetry without. The dollar sign is necessary for statements, but `Invoke-WebRequest -Uri {…}` is not a statement but an expression. See https://stackoverflow.com/questions/47274532/difference-between-and.